### PR TITLE
change of sample name and identifier in device information

### PR DESCRIPTION
**Pull request by RonHildebrandt**
prepares the changes which will come due to the nexus def. of [PR284](https://github.com/FAIRmat-NFDI/nexus_definitions/pull/284) in NeXus def.

### CODE REVIEW

**Comment by RonHildebrandt** [comment]
Indeed, it is also important to note which specific commit. [This one](3ceecffd5a211f465323d8961d0ee6b8fe3c4f29) was it tested for now.

**Review by lukaspie — CHANGES_REQUESTED** [review]
You need to update the `tests/data/example.nxs` file in order for the tests to pass (i.e., that one should be the new file that you generate after these changes).

File: pynxtools_raman/reader.py [hunk 1]
```diff
@@ -28,14 +28,14 @@
 
 CONVERT_DICT = {
```
[the remark this answers is not in the document]
Comment (RonHildebrandt) [reply pynxtools_raman/reader.py:29]: okay, thanks. Ill noted it and will address this later.

File: tests/data/eln_data.yaml [hunk 2]
```diff
@@ -68,7 +68,8 @@ Instrument:
     detector_type: CCD
     device_information:
       model: DU970_BV
-      identifier: "Serial Number: 6358"
+      identifier:
+        identifier: "Serial Number: 6358"
```
[the remark this answers is not in the document]
Comment (RonHildebrandt) [reply tests/data/eln_data.yaml:72]: oh, this was added. right. Thanks - remembered this discussion as you mentioned it.

**Comment by RonHildebrandt** [comment]
The `tests/data/example.nxs` file is relevant for this at all? Was totally not aware of that. Okay., ill test it.

**Pull request merged**